exercises/simple-cipher/simple_cipher_test.exs

```diff
@@ -0,0 +1,117 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("simple_cipher.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule SimpleCipherTest do
+  use ExUnit.Case
+
+  describe "random key generation" do
+    #@tag :pending
+    test "random key is all lowercase letters" do
+      key = SimpleCipher.key
+
+      assert key != nil
```

Comment:
I think this test is kind of superfluous since the regexp on the next line will only match if there is one or more letter, right? Think we still need this?